### PR TITLE
Ability to download in ridelog view

### PR DESCRIPTION
**Pull request by RTeran**
Ability to download in ridelog view has been added

### CODE REVIEW

File: trailforks-export.js [hunk 1]
```diff
@@ -10,17 +10,17 @@ function downloadString(filename, text) {
 }
 
 //checks for problems, alerts user
-function errCheck(pageString) {
+function errCheck(pageStr) {
```
Comment (cosmocatalano) [inline trailforks-export.js:13]: Assuming you changed `pageString` to `pageStr` to avoid potential conflict?

**Review by cosmocatalano — COMMENTED** [review]
Thanks for this, and sorry for the delay! Assuming just a quick check in on a variable name change, otherwise seems good!

File: trailforks-export.js (hunk 1 above)
Comment (RTeran) [reply trailforks-export.js:13]: Exactly 👌

**Review by cosmocatalano — APPROVED** [review]
Thanks! And sorry this took forever.

**Pull request merged**